.eslintrc.js

```diff
@@ -262,6 +262,7 @@ module.exports = {
     {
       files: [
         '**/__snapshots__/*.snap',
+        'app/scripts/controllers/app-state.test.js',
```

Comment:
It looks like this needs to be listed under the `excludedFiles` entry of the block above here. That's probably why the linter converted the test to use functions - it's a mocha rule, and that's still getting applied.

Reply:
Probably should call out more explicitly that this list needs to be in-sync with the list in the previous block. I see there are other discrepancies as well

Reply:
I could sync that up, but I think the formatting has to be changed to lambda notation in each test file manually, so it might be better left for a separate PR/changeset.

Reply:
Oh yeah definitely, I agree